### PR TITLE
Roster update 01 22

### PR DESCRIPTION
**Pull request by gonuke**
Updates to the personnel for Jan 2022.

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -205,6 +186,21 @@
...
+    employer_url: http://ners.engin.umich.edu
+  - name: Matthew Klebenow
+    url:
+    year: 2013
+    role: BS Engineering Physics
+    employer: EPIC Systems
+    employer_url:
   - name: Bradly Laufenberg
     year: 2021
     role: BS Nuclear Engineering
```
Comment (NybergWISC) [inline _data/people.yml:207]: Brad works at INL now as part of the Naval Nuclear Lab

**Comment by gonuke** [comment]
Thanks @NybergWISC - it's ready for another review

**Comment by gonuke** [comment]
BTW - PR #197 includes an update to the README with a robust command to launch a docker image to test

File: _data/people.yml [hunk 2]
```diff
@@ -169,30 +161,15 @@
     role: PhD
     employer: University of Illinois
     employer_url: http://npre.illinois.edu
+  - name: ShaiAnthony Huff
```
Comment (lewisgross1296) [inline _data/people.yml:164]: Is there supposed to be a space here between Shai and Anthony? (not sure)

File: _data/people.yml [hunk 3]
```diff
@@ -205,9 +182,31 @@
...
+    role: Undergraduate intern
+  - name: Brian Kiedrowski
+    url: http://www.engin.umich.edu/ners/people/faculty/brian-kiedrowski
+    year: 2009
+    role: PhD
+    employer: University of Michigan
+    employer_url: http://ners.engin.umich.edu
+  - name: "Kalin R. Kiesling"
+    role: PhD
+    year: 2020
```
Comment (opotowsky) [inline _data/people.yml:196]: Typo -- 2021 graduation, correct?

**Comment by gonuke** [comment]
This is ready for another look/merge (@opotowsky ?)

**Pull request merged**